src/main/java/com/revaturelabs/ask/user/UserServices.java

```diff
@@ -72,7 +72,7 @@ public User update(User user) throws UserNotFoundException, UserConflictExceptio
       try {
         updatedUser = userRepo.save(user);
       } catch (DataIntegrityViolationException e) {
-        throw new UserConflictException();
+        e.printStackTrace();
```

Comment:
Where will this stack trace be printed? This is not an appropriate solution.  Change the exception message, but do throw the exception